### PR TITLE
Scriptss3

### PR DESCRIPTION
**Pull request by HelgaStock**
created scripts setup_s3, upload-src
created lambda-monitor.tf
created folder monitor & server-monitor.py

### CODE REVIEW

File: architecture/lambda-monitor.tf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+resource "aws_lambda_function" "server_monitoring-function" {
+  function_name = "server_monitoring_function"
+  filename      = "architecture/build/monitor.zip"
+  role          = "arn:aws:iam::534283426081:role/LabRole"
```
Comment (fabianschmauder) [inline architecture/lambda-monitor.tf:4]: I would replace the part with a ref to your terraform account id.

File: architecture/lambda-monitor.tf [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  handler       = "monitor.handler"
+  timeout       = 300
+  runtime       = "python3.9"
+
+  environment {
+    variables = {
+      MONITOR_TABLE_NAME = aws_dynamodb_table.SD-source-table.name
```
Comment (fabianschmauder) [inline architecture/lambda-monitor.tf:11]: i would rename your table to lower case. (sd-source-table) But it is not a real problem

File: monitor/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+fastapi==0.85.1
```
Comment (fabianschmauder) [inline monitor/requirements.txt:1]: should monitor be a lambda function or a server. 
If it is a lambda function than you do not need fast api and uvicorn.

If it is a server on ec2 you dont have a handler like in server-monitor.py

File: architecture/DBsource.tf [hunk 4]
```diff
@@ -1,5 +1,5 @@
-resource "aws_dynamodb_table" "SD-source-table" {
-  name           = "SD-source-table"
+resource "aws_dynamodb_table" "sd-source-table" {
+  name           = "sd-source-table"
   read_capacity  = 500
```
Comment (HelgaStock) [inline architecture/DBsource.tf:3]: adjust read_capacity to 20
adjust write_capacity to 20

**Review by HelgaStock — COMMENTED** [review]
all changes implemented as suggested

**Review by HelgaStock — COMMENTED** [review]
all changes implemented, testate files removed from GitHub directory

**Review by HelgaStock — COMMENTED** [review]
branch ready for merge

**Pull request merged**